webcompat/templates/shared/svg-icons.html

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think we want to remove this XML declaration, pretty sure it's not needed when embedding in HTML.

Reply:
#2311